NodeJS/nextjs/next.config.js

```diff
@@ -0,0 +1,13 @@
+/** @type {import('next').NextConfig} */
```

Comment:
Is this really necessary? (I really don't know what this does hahaha)

Reply:
Yeah, really odd. This is part of the nextjs boilerplate.

As I understand it, this JSDoc helps provide type information and documentation for the configuration object, making it easier to work with and ensuring type safety in the code (although this is a js file).

Without it my editor fails to suggest completion.

More here: https://nextjs.org/docs/pages/api-reference/next-config-js